dev/src/dev/debug_qp.clj

```diff
@@ -1,26 +1,71 @@
 (ns dev.debug-qp
+  "TODO -- I think this should be moved to something like [[metabase.test.util.debug-qp]]"
   (:require [clojure.data :as data]
             [clojure.pprint :as pprint]
             [clojure.string :as str]
-            [clojure.test :refer :all]
             [clojure.walk :as walk]
             [medley.core :as m]
             [metabase.mbql.schema :as mbql.s]
             [metabase.mbql.util :as mbql.u]
             [metabase.models :refer [Field Table]]
             [metabase.query-processor :as qp]
             [metabase.query-processor.reducible :as qp.reducible]
-            [metabase.test :as mt]
             [metabase.util :as u]
             [toucan.db :as db]))
 
-;; see docstring for `process-query-debug` for descriptions of what these do.
+;;;; [[add-names]]
```

Comment:
intended?

Reply:
yeah. https://guide.clojure.style/#four-semicolons-for-heading-comments and https://guide.clojure.style/#document-references